lib/companies/jobs.ex

```diff
@@ -18,7 +18,10 @@ defmodule Companies.Jobs do
 
   """
   def all do
-    Repo.all(Job)
+    (j in Job)
+    |> from()
+    |> where([j], is_nil(j.removed_pending_change_id))
```

Comment:
In addition to check for non-deleted jobs, should we check for jobs belonging to non-deleted companies?

Reply:
Yes this seems live a valid concern. If the company is soft-deleted, its jobs should *not* show up as well